docs/Guides/Automate/WebhookAction.md

```diff
@@ -0,0 +1,167 @@
+# Use the trigger a webhook action
+
+You can take action in your systems based on events in the Vendasta platform by using automations. With any automation you can add the `Trigger a webhook` step to be notified when something happens. 
+
+## Overview
+
+In this guide we will create an automation, listen for the event and retrieve details of the affected records.
+
+## Prerequisites
+
+This guide assumes you are familiar with creating automations within the platform. If you are not familiar you should [learn more about automations](https://support.vendasta.com/hc/en-us/sections/4406950706583-Automations) first.
+
+
+## Step 1: Automation setup
+
+In Partner Center go to the automations screen and create an automation based on the trigger and criteria that you are interested in.
+
+Add the `Trigger a webhook` step.
+
+![](webhookAddPart1.png)
+
+The **webhook URL** will be an endpoint that you have built in step 2. It will likely be one of your web servers but could also be a 3rd party system like Zapier. 
```

Comment:
step 2 [of the automation] maybe?

Reply:
I think `step 2` in this case refers to step 2 of this guide

Reply:
Oh I see!! Yes maybe do that instead!